pkg/features/kube_features.go

```diff
@@ -62,6 +62,13 @@ const (
 	//
 	// Enables Kueue visibility on demand
 	VisibilityOnDemand featuregate.Feature = "VisibilityOnDemand"
+
+	// owner: @yaroslava-serdiuk
+	// kep: https://github.com/kubernetes-sigs/kueue/issues/1283
+	// alpha: v0.6
+	//
+	// Enable priority sorting within the cohort.
+	PrioritySortingWithinCohort featuregate.Feature = "PrioritySortingWithinCohort"
```

Comment:
Since we want this to be a long term option, it should be in here https://github.com/kubernetes-sigs/kueue/blob/main/apis/config/v1beta1/configuration_types.go

Reply:
As discussed offline, agreed to leave feature gate and try to find better solution for initial problem.